.github/workflows/ecs.yml

```diff
@@ -30,6 +30,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: custom/plugins/${{ inputs.extensionName }}
+      - name: ECS cache
+        uses: actions/cache@v3
+        with:
+          path: var/cache/cs_fixer
+          key: ${{ runner.OS }}-${{ github.repository }}-ecs-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.OS }}-${{ github.repository }}-ecs-
       - name: Run ECS
         working-directory: custom/plugins/${{ inputs.extensionName }}
         run: ${{ github.workspace }}/vendor-bin/cs-fixer/vendor/bin/ecs check
```

Comment:
Can you set here `SHOPWARE_TOOL_CACHE_ECS` as env to `${{ github.workspace }}/var/cs_fixer` and then we cache that folder? It's more stable ;)

Reply:
Done